docs/src/5-prestashop-billing/5-use-cases/README.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Use Cases
+---
+
+[[toc]]
+
+# Concepts
+
+Prestashop Billing currently supports four types of pricing models:
+
+- Flat fee
+- Stairstep
+- Usage-based
+- Charge at term
+
+Each pricing model has its advantages and suitability depending on the nature of the SaaS product, target audience, and the specific needs of the customers
+
+### Flat Fee
+
+In the flat fee pricing model, customers are charged a fixed and consistent amount for using the SaaS product or service. Regardless of the usage volume or frequency, the cost remains the same. This model offers simplicity and predictability, making it easy for customers to understand and budget their expenses. It is particularly suitable for users who require unlimited access to the SaaS product without worrying about usage-based charges.
+
+**_Example:_** A cloud storage SaaS offers a flat fee pricing model of 10€ per month for 100GB of storage space. Regardless of whether the customer uses 50GB, 100GB, or even less than 100GB, the monthly cost remains fixed at 10€.
```

Comment:
**todo**: pourrais-tu stp remplacer cet exemple par celui là 

```
Example: A mass data management SaaS offers a flat fee pricing model of 30€ per month for a basic plan and 60€ for a plan with more features.
```

Après review de la page notion https://www.notion.so/prestashopcorp/What-is-PS-Billing-90e9f1357ec84fde8ec908f21b83eb59 on préfère un exemple qui ne parle pas de quantité par peur que ce soit confus

Reply:
Oui pas de soucis, j'ai repris ce qu'il y avait dans le ticket ;) 

Effectivement ça sera plus parlant comme ça :)